driverconfig/config.yaml

```diff
@@ -186,9 +186,9 @@ csiSideCars:
   - name: attacher
     images:
       - version: v121
-        tag: k8s.gcr.io/sig-storage/csi-attacher:v4.0.0
+        tag: k8s.gcr.io/sig-storage/csi-attacher:v4.2.0
       - version: v122
-        tag: k8s.gcr.io/sig-storage/csi-attacher:v3.5.0
+        tag: k8s.gcr.io/sig-storage/csi-attacher:v4.2.0
```

Comment:
@nitesh3108, why are we changing the sidecar versions for v1.22. Please revert this change for v1.22, as this version is not supported this release as well as last release
Same comment for all sidecars below

Reply:
Can't we keep all sidecar version to latest, irrespective we enable 1.22 support or not ?

Reply:
No, we keep at those versions which was tested and we dont keep updating the sidecar versions unless qualified for each K8s versions